File: 백준/Bronze/11050. 이항 계수 1/이항 계수 1.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());
        int N = Integer.parseInt(st.nextToken());
        int K = Integer.parseInt(st.nextToken());
        K = Math.min(K, N - K);
        int ans = 1;
        for(int i = N; i > N - K; i--) {
            ans *= i;
        }
        for(int i = 1; i <= K; i++) {
            ans /= i;
        }
        System.out.println(ans);
    }
}

File: 백준/Bronze/11050. 이항 계수 1/이항 계수 1.js
let [N, K] = require('fs').readFileSync('/dev/stdin').toString().trim().split(' ').map(Number);
K = Math.min(K, N - K);
let ans = 1;
for(let i = N; i > N - K; i--) {
  ans *= i;
}
for(let i = 1; i <= K; i++) {
  ans /= i;
}
console.log(ans);